src/applications/vaos/components/ConfirmedAppointmentListItem.jsx

```diff
@@ -97,8 +98,37 @@ export default function ConfirmedAppointmentListItem({
         )}
       </div>
 
+      {!allowCancel &&
+        !canceled && (
+          <div className="vads-u-margin-top--2">
+            <button
+              onClick={() => exportICSFile(appointment, facility)}
+              aria-label="App appointment to calendar"
+              className="vaos-appts__cancel-btn va-button-link vads-u-margin--0 vads-u-flex--0"
+            >
+              Add to calendar
+              <span className="sr-only">
+                {' '}
+                on {getAppointmentDate(appointment)}
+              </span>
+            </button>
+          </div>
+        )}
+
       {allowCancel && (
         <div className="vads-u-margin-top--2">
+          <button
```

Comment:
I think we could avoid having two of these buttons in our component markup with some reworking of when the outer div is shown.

Reply:
I'm not sure how since the "allowCancel" flag checks for appointments types not equal to CC and appointments not equal to video visit.

Reply:
We'd probably have to tweaks some of those checks. I'm not sure of exactly what it would look like without playing around with it, so I don't think we should hold up this PR for it or anything.

Reply:
Ok.